setup.py

```diff
@@ -36,18 +36,18 @@ def read_file(filename: str) -> str:
         'flake8',
         'flake8-bugbear',
         'flask',
-        'google-api-python-client==2.10.0',
-        'google-auth==1.27.0',
-        'google-cloud-logging==3.1.1',
-        'google-cloud-pubsub==2.3.0',
-        'google-cloud-secret-manager==2.2.0',
-        'google-cloud-storage==1.25.0',
+        'google-api-python-client>=2.10.0,<3',
+        'google-auth>=1.27.0,<2',
+        'google-cloud-logging>=3.1.1,<4',
+        'google-cloud-pubsub>=2.3.0,<3',
+        'google-cloud-secret-manager>=2.2.0,<3',
+        'google-cloud-storage==1.25.*',
```

Comment:
Did you mean to pin the minor version here?

Reply:
That's what I'd usually do, but this is copied directly from `hail` for maximum compatibility.

Reply:
Updated.